config/puma.rb

```diff
@@ -42,3 +42,11 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+pidfile "/tmp/puma.pid"
```

Comment:
@ereslibre for what it's worth, take a look at what we are doing in Portus: https://github.com/mssola/Portus/blob/2061d5d1e78bc20f362863c4699b6d18e854c00d/config/puma.rb. That being said, I'm not sure how the `pidfile` is useful inside of a container ...